maxmind.json

```diff
@@ -36,67 +37,79 @@
         "wheel": [
             {
                 "module": "certifi",
-                "input_file": "wheels/py2/certifi-2019.11.28-py2.py3-none-any.whl"
-            },
-            {
-                "module": "chardet",
-                "input_file": "wheels/py2/chardet-3.0.4-py2.py3-none-any.whl"
+                "input_file": "wheels/shared/certifi-2019.11.28-py2.py3-none-any.whl"
```

Comment:
Since I'm not familiar with these wheels change, @pzhou-splunk Could you help check if these wheels change look reasonable?